### PR TITLE
Adding batch buttons back to my work listing.  Allows for deleting mu…

### LINKED ISSUE #1097: Add Batch controls Back into my files listing
Batch controls were out of scope for #999 so they were changed to TODOs.

Add the batch controls back into the dashboard.

The Add To Collection flow should still work for works, so test that flow.  If it does not work, create and additional ticket for making that work.

related ticket: #1096, #1003

Depends on #1093

### PR DESCRIPTION
**Pull request by cam156**
…ltiple works. fixes #1097

Removed todo and put the batch select boxes back.  Delete is working and there are tickets in for the collection show page.

screen shot 2015-05-12 at 8 00 47 am

### CODE REVIEW

File: app/views/my/_sort_and_per_page.html.erb [hunk 1]
```diff
@@ -5,10 +5,6 @@
 
   <% if on_my_files? %>
     <div class="batch-toggle">
-      <% session[:batch_edit_state] = "on" %>
```
Comment (jcoyne) [inline app/views/my/_sort_and_per_page.html.erb]: :+1:

File: spec/features/browse_dashboard_files_spec.rb [hunk 2]
```diff
@@ -63,10 +63,11 @@
       end
     end
 
-    it "should allow me to edit works in batches", skip: 'Not yet implemented' do
+    it "should allow me to delete works in batches",  :js => true do
```
Comment (jcoyne) [inline spec/features/browse_dashboard_files_spec.rb]: What would @mjgiarlo say?

Comment (cam156) [inline spec/features/browse_dashboard_files_spec.rb]: @jcoyne we are not planning to allow for batch edit of works at the moment, so I though no reason to leave the test hanging around. What part of this line is @mjgiarlo going to comment on?

Comment (jcoyne) [inline spec/features/browse_dashboard_files_spec.rb]: :hash: :rocket:

Comment (cam156) [inline spec/features/browse_dashboard_files_spec.rb]: I'll remove it.  I do not even see them...

**Pull request merged**